_data/api/pro-sante-connect.md

```diff
@@ -0,0 +1,70 @@
+
+      Si vous cherchez à contacter l'équipe Pro Santé Connect, vous pouvez [contacter l'Agence Numérique en Santé](https://esante.gouv.fr/assistance)
+
+      <Button href="/rechercher-api">Ou vous pouvez rechercher une autre API</Button>
+  - who:
+      - Une entreprise
+    is_eligible: 0
+    description: |
+      Seuls les éditeurs de logiciels proposant une solution web adressée aux professionnels de santé peuvent demander à se raccorder à PRO Santé Connect.
+      <NextSteps />
+      <Button href="https://franceconnect.gouv.fr/partenaires">Remplir une demande</Button>
+  - who:
+      - Une collectivité ou une administration
+    is_eligible: 1
+    description: |
+      Vous pouvez réaliser une demande d'accès à l'API. 
+      
+  - Un éditeur de logiciels
+    is_eligible: 0
+    description: |
+      Tout éditeur proposant une solution web adressée aux professionnels de santé et qui souhaite pouvoir authentifier des professionnels de santé peut demander à se raccorder à PRO Santé Connect. 
+
+      <NextSteps />
+      <Button href="mailto:prosanteconnect.editeurs@esante.gouv.fr">Remplir une demande</Button>
+partners:
+  - Ministère des Solidarités et de la Santé (MSS)
+producer: ans
+keywords:
+  - Authentification
+  - Identification
+  - SSO
+  - Santé
+  - Professionnels
+  - Editeurs
+themes:
+  - Santé
+contact_link: prosanteconnect.editeurs@esante.gouv.fr
+doc_tech_external: https://integrateurs-cps.asipsante.fr/pages/prosanteconnect/documentation-fs
+last_update: 22/03/2021
+---
+
+L‘API Pro Santé connect permet à des administrations, des éditeurs de logiciels ou des entreprises d’ajouter un bouton Pro Santé Connect, pour recueillir des données d’identité fiables et ainsi identifier les utilisateurs de leur service en ligne (en vue d’une entrée en relation ou connexion).
+
+### A quoi ca sert ?
+
+PRO Santé Connect est un fournisseur d’identité (FI) au standard OpenID. Il accepte comme « credential » d’authentification la carte CPS et une e-carte CPS qui est une application téléphone. L’assertion d’authentification OpenIDConnect « remonte » au fournisseur de service (FS) demandeur de l’authentification les traits d’identités classiques plus l’identité sectorielle (profession, savoir-faire, situation d’exercice, …) afin que celui-ci puisse, au-delà de la simple authentification, gérer tout ou partie de son contrôle d’accès . Ces éléments sont fournis par [notre annuaire](https://annuaire.sante.fr) et incorporé dans le jeton OpenIdConnect.
+                                                                                                                          |
+
+### Comment ça marche ?
+
+**L'identité pivot**.
+
+| Type       | Données                      |
+| --------- | -------------------------------- |
+| Données d'identification       | nom, prénom, idNat (Adeli ou RPPS) |
```

Comment:
I would split it this way
| Identifiant unique (idNat) | Adeli ou RPPS |
| état civil | nom et prénom |